src/Avalonia.Controls/Platform/IWin32OptionsTopLevelImpl.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Avalonia.Metadata;
+using Avalonia.Platform;
+
+namespace Avalonia.Controls.Platform
+{
+    [PrivateApi]
+    public interface IWin32OptionsTopLevelImpl : ITopLevelImpl
+    {
+        public delegate (uint style, uint exStyle) CustomWindowStylesCallback(uint style, uint exStyle);
```

Comment:
Those delegates are still accessible despite the `PrivateApi` but I don't know whether it's the expected behavior or not. All `PrivateApi` members and nested types normally become `internal` in the ref assemblies, but delegates aren't affected currently. @kekekeks

Reply:
Callback delegate should be declared in Win32SpecificOptions

Reply:
Moved.